src/site/content/en/blog/wakelock/index.md

```diff
@@ -72,9 +71,9 @@ Of course, there are plenty of other use cases:
 | ------------------------------------------ | ---------------------------- |
 | 1. Create explainer                        | N/A                          |
 | 2. Create initial draft of specification   | [Complete][spec-ed]          |
-| **3. Gather feedback and iterate design**  | [**In Progress**](#feedback) |
-| **4. Origin trial**                        | [**In Progress**][ot]        |
-| 5. Launch                                  | Not Started                  |
+| 3. Gather feedback and iterate design      | [Complete](#feedback)        |
+| 4. Origin trial                            | [Complete][ot]               |
```

Comment:
Is one of these links formatted wrong? One has a pound sign (#). The other doesn't. (I put my links in line, so I don't know.)

Reply:
Also, if the OT is complete, this doesn't need a link.

Reply:
The one goes to a fragment in the text, the other uses the markdown link lookup feature. Since the OT link is no longer needed, I'll just remove it :-)

Reply:
Fragments are only supported in Chromium forks. This is a platform site, not Chromium. Please change your links.

Reply:
In this case it's really just a regular element fragment link that points to the Feedback heading. Test it here: https://deploy-preview-3310--web-dev-staging.netlify.app/wakelock/#feedback.